configure.ac

```
@@ -520,7 +520,8 @@ OCAML_CC_VENDOR
 
 AS_CASE([$ocaml_cv_cc_vendor],
   [xlc-*],
-    [outputobj='-o $(EMPTY)'; gcc_warnings="-qflag=i:i"], # all warnings enabled
+    [outputobj='-o $(EMPTY)'; gcc_warnings="-qflag=i:i"; # all warnings enabled
+     CPP="$(CC) -E -qnoppline"], # suppress incompatible XLC line directives
   [msvc-*],
     [outputobj=-Fo; CPP="cl -nologo -EP"; gcc_warnings=""],
```

```suggestion
    [outputobj=-Fo; gcc_warnings=""],
```
(`CPP` is already for MSVC in 450-452)

---

> It should be relatively simple for to get configure.ac to test whether __attribute((aligned... is available which would solve this permanently for exotic C compilers and also move the error detection out of the C headers and into configure where it belongs.

I'll try to do it.